src/dex-ui/pages/Governance/ManageVotingPower/ManageVotingPower.tsx

```diff
@@ -0,0 +1,80 @@
+import { Text, VStack, HStack, Button } from "@chakra-ui/react";
+import { Color } from "../../../../dex-ui-components/themes";
+import { useTokenBalance } from "../../../hooks";
+import { LightningBoltIcon, TwoLayerSettingsIcon, SwapIcon } from "../../../../dex-ui-components/base/Icons";
+import { PageHeader } from "../../../layouts";
+import { GovernanceTokenId } from "../../../services";
+import { VotingPowerCard } from "./VotingPowerCard";
+
+export const ManageVotingPower = () => {
+  const { data: godTokenBalance, isLoading } = useTokenBalance({ tokenId: GovernanceTokenId });
+  function handleClickSwapButton() {
+    console.log("Swap Button clicked");
+  }
+
+  function handleClickManageButton() {
+    console.log("Manage Button clicked");
+  }
+
+  return (
+    <PageHeader
```

Comment:
The `PageHeader` Component was intended to solely be used for the page title and an action button:

Example:
<img width="971" alt="Screen Shot 2023-04-06 at 3 11 01 PM" src="https://user-images.githubusercontent.com/54907098/230472913-41db6bf0-c5e1-4074-bb6b-918a7ade175a.png">
```
<PageHeader
     leftContent={[<Text textStyle="h2">DAOs</Text>]}
     rightContent={[<PrimaryHeaderButton name="Create new DAO" route={Paths.DAOs.CreateDAO} />]} />
```
The `leftContent` and `rightContent` are ambiguous so I can see how this is misleading. I suggest wrapping these voting power metrics in a standalone `Flex` component so the spacing is not accidently impacted by changes in the PageHeader component.

Reply:
Done